docs/gettinginvolved/cluster-setup.rst

```diff
@@ -199,6 +171,79 @@ Or you can run, for example, the acceptance tests against the created cluster:
      --flocker-version='' \
      flocker.acceptance.obsolete.test_containers.ContainerAPITests.test_create_container_with_ports
 
+==============================
+Adding containers and datasets
+==============================
+To make it easier to reuse a test cluster, and test under different configurations, Flocker provides a tool to create a certain number of datasets and containers per node.
+It is called like this:
```

Comment:
Happy to reword it @davidoliverSP2 . I am not great getting the best way of explaining stuff in English - all I can learn on that is more than welcome!